src/TaskRunner/Commands/ToolCommands.php

```diff
@@ -1203,4 +1210,29 @@ private function getPackagePropertyFromComposer($package, $prop = 'version')
         }
         return false;
     }
+
+    /**
+     * Return the QA API BASIC AUTH from token or from questions.
+     *
+     * @return string
+     *   The Basic auth or empty string if fails.
+     */
+    public function getQaApiBasicAuth(): string
+    {
+        $auth = getenv('QA_API_BASIC_AUTH');
+        if (empty($auth)) {
+            $this->say('Missing env var QA_API_BASIC_AUTH, asking for access.');
+            if (empty($user = $this->ask('Please insert your username?'))) {
+                $this->writeln('<error>The username cannot be empty!</error>');
+                return '';
+            }
+            if (empty($pass = $this->ask('Please insert your password?', true))) {
+                $this->writeln('<error>The password cannot be empty!</error>');
+                return '';
+            }
+            $auth = base64_encode("$user:$pass");
```

Comment:
Add feedback with indication to include/store the token in the environment vars?